packages/monitor-v2/src/monitor-polymarket/common.ts

```diff
@@ -482,6 +489,10 @@ export const initMonitoringParams = async (env: NodeJS.ProcessEnv): Promise<Moni
   if (!env.CHAIN_ID) throw new Error("CHAIN_ID must be defined in env");
   const chainId = Number(env.CHAIN_ID);
 
+
+  if (!env.POLYMARKET_API_KEY) throw new Error("POLYMARKET_API_KEY must be defined in env");
+  const polymarketApiKey = env.POLYMARKET_API_KEY;
```

Comment:
only thing id say is we dont necessarily need to crash if not defined, should be optional. but not a huge deal now